go/cmd/vtctldclient/command/schema.go

```diff
@@ -141,7 +141,7 @@ func commandApplySchema(cmd *cobra.Command, args []string) error {
 		AllowLongUnavailability: applySchemaOptions.AllowLongUnavailability,
 		DdlStrategy:             applySchemaOptions.DDLStrategy,
 		Sql:                     parts,
-		SkipPreflight:           applySchemaOptions.SkipPreflight,
+		SkipPreflight:           true,
```

Comment:
same comments I made in https://github.com/vitessio/vitess/pull/10717#discussion_r928711419, but for this flag

Reply:
:+1:

Reply:
Comments addressed